sys/net/link_layer/ng_netdev_eth/ng_netdev_eth.c

```diff
@@ -99,37 +97,37 @@ static int _send_data(ng_netdev_t *netdev, ng_pktsnip_t *pkt)
     int nsent, to_send;
     ng_netdev_eth_t *dev = (ng_netdev_eth_t *)netdev;
 
-    DEBUG("ng_netdev_eth: send data ");
+    LOG_DEBUG("ng_netdev_eth: send data ");
 
     if (pkt == NULL) {
         return -EFAULT;
     }
 
     if ((dev == NULL) || (netdev->driver != &ng_netdev_eth_driver)) {
-        DEBUG("[wrong device descriptor]\n");
+        LOG_ERROR("[wrong device descriptor]\n");
```

Comment:
Either making it too distinct messages or making one of them the level of the other.